exercises/practice/knapsack/.meta/example.nim

```diff
@@ -0,0 +1,8 @@
+type Item = tuple[weight: int, value: int]
+
+proc maximumValue*(maximumWeight: int, items: openArray[Item]): int =
+    var dp = newSeq[int](maximumWeight + 1)
```

Comment:
```suggestion
    var dp = newSeq(maximumWeight + 1)
```
I don't think this is needed

Reply:
Great, happy to make the change.  
Do you happen to have a link to a resource where I can learn when I do and do not have to specify the template parameter?

Reply:
Now the five jobs of the CI fail. I will roll this back.
Maybe it's a version thing because on my computer with nim-2.0.0 I did indeed not need the template parameter.

Reply:
Weird!
I will try to take a look in the next couple of days

Reply:
@ynfle did you have the time to look at it?

Reply:
It was a mistaken suggestion. The `[int]` tells the compiler what is the type of the seq. It can not infer the type of the elements in the seq from the code because none of elements are given, yet